ehealthid-rp/src/main/java/com/oviva/ehealthid/relyingparty/ws/ui/Pages.java

```diff
@@ -13,14 +15,29 @@ public Pages(TemplateRenderer renderer) {
     this.renderer = renderer;
   }
 
-  public String selectIdpForm(List<IdpEntry> identityProviders) {
+  public String selectIdpForm(List<IdpEntry> identityProviders, String language) {
```

Comment:
```suggestion
  public String selectIdpForm(List<IdpEntry> identityProviders, Locale locale) {
```
Any reason to pass it untyped?